ext/vscode/docs/commands.md

```diff
@@ -0,0 +1,70 @@
+# VS Code Extension Commands
+
+This document describes the commands provided by the Azure Developer CLI Visual Studio Code extension.
+
+## Command List
+
+Most commands are available in the Command Palette (Ctrl+Shift+P or Cmd+Shift+P) and are prefixed with "Azure Developer CLI (azd)".
+
+## Utility Commands
+
+Some commands are not directly available in the command palette but are provided for use in other VS Code features.
+
+### azure-dev.commands.getDotEnvFilePath
+
+This command retrieves the path to the `.env` file for a specified Azure Developer environment or for the default environment if none is specified. This is useful for VS Code configurations where you need to access environment variables from an Azure Developer environment.
```

Comment:
Here and throughout the document, you should say "Azure Developer CLI environment" instead of "Azure Developer environment".